src/components/AutoCompleteSuggestions/index.tsx

```diff
@@ -53,11 +53,13 @@ function AutoCompleteSuggestions({measureParentContainer, ...props}) {
 
     return (
         Boolean(width) &&
-        ReactDOM.createPortal(<View style={StyleUtils.getBaseAutoCompleteSuggestionContainerStyle({left, width, bottom})}>{componentToRender}</View>, document.querySelector('body'))
+        ReactDOM.createPortal(
+            <View style={StyleUtils.getBaseAutoCompleteSuggestionContainerStyle({left, width, bottom})}>{componentToRender}</View>,
+            document.querySelector('body') as Element,
```

Comment:
`document.querySelector('body')` can be null, shouldn't we check that it is defined instead of asserting?